symphony/app/fbcnms-projects/magmalte/app/theme/default.js

```diff
@@ -211,6 +211,9 @@ export default createMuiTheme({
     },
     blueGrayDark: colors.primary.brightGray,
   },
+  typography: {
+    ...typography,
+  },
   overrides: {
     MuiAppBar: {
       colorPrimary: {
```

Comment:
Not sure why i'm unable to comment on code below. Can you perhaps add the primary and secondary text typography styles in  MuiListItemText, that way we can avoid setting body3 variant for primary and secondary through out the screens.

Reply:
Most changes LGTM. It would be great if you could squash the commits before submitting. It helps to see the full picture. When going through the list of commits , it is hard to read when one commit removes the changes from previous commit.

Reply:
Thanks for the suggestion and review. 

I've updated the `MuiListItemText` to default to `body3` and subsequently removed those designations from the list in `subscriberDetailConfig.js`